nicegui/client.py

```diff
@@ -84,6 +85,12 @@ def is_auto_index_client(self) -> bool:
         """Return True if this client is the auto-index client."""
         return self is self.auto_index_client
 
+    @staticmethod
+    def current_client() -> Optional[Client]:
+        """Returns the current client if obtainable from the current context."""
+        from .context import get_client
```

Comment:
Should the import not be made at the top of the file?

Reply:
Changed. To minimize circular import issues I tend to use such "one-shot" usages into methods.